### PR TITLE
Empty state story

### PR DESCRIPTION
**Pull request by Oluwadaminiola**

### CODE REVIEW

File: web/ui/dashboard/src/stories/empty-state/Empty-state.stories.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		},
+        buttonText: {
+			control: { type: 'string' }
+		},
+		type: {
+			options: ['normal', 'table'],
+			control: { type: 'select' },
+			defaultValue: 'normal'
+		},
+        class: {
```
Comment (horlah) [inline web/ui/dashboard/src/stories/empty-state/Empty-state.stories.ts:25]: This is not needed here

Comment (horlah) [reply web/ui/dashboard/src/stories/empty-state/Empty-state.stories.ts:25]: We're meant to show the components in their natural state

**Review by horlah — CHANGES_REQUESTED** [review]

**Comment by horlah** [comment]
Screenshot 2022-08-16 at 3 03 50 PM

I can't edit controls here.

**Comment by horlah** [comment]
I might have misdirected about removing the class property in the stories you created. Leaving the class there will allow us to test some styles on the component while testing or before applying it to the main component; since we're using Tailwind everywhere.

What do you think @Oluwadaminiola

**Comment by Oluwadaminiola** [comment]
> I might have misdirected about removing the class property in the stories you created. Leaving the class there will allow us to test some styles on the component while testing or before applying it to the main component; since we're using Tailwind everywhere.
> 
> What do you think @Oluwadaminiola

yeah, I kind of agree with this, although due to the fact that most component stories are just standalone, the use cases for using the class property would be a bit limited, hence why I didn't necessarily add them to every story

**Pull request merged**